bin/ntlink_pair.py

```diff
@@ -154,6 +158,35 @@ def print_directed_graph(graph, out_prefix, scaffolds):
 
         outfile.write("}\n")
 
+    @staticmethod
+    def print_directed_gfa2_graph(graph, out_prefix, scaffolds):
+        "Prints the directed scaffold graph in GFA2 format"
+
+        out_graph = out_prefix + ".scaffold.gfa"
+        outfile = open(out_graph, 'w')
+        print(datetime.datetime.today(), ": Printing graph", out_graph, sep=" ", file=sys.stdout)
+
+        outfile.write("H\tVN:Z:2.0")
+
+        graph_info = "graph [scaf_num={}]\n".\
+            format(NtLink.get_largest_ntlink_scaffold_id(scaffolds))
+        outfile.write(graph_info)
+
+        for node in graph.vs():
+            node_str = "S\t{scaffold}\t{length}".\
+                format(scaffold=node['name'], length=scaffolds[node['name'][:-1]].length)
+            outfile.write(node_str)
```

Comment:
Think you're missing newline characters (will probably catch that when you're adding tests anyway :) )

Reply:
yep, I just started using some toy examples and porting outputs to `gfaviz` and `gfatools` to make sure the output is correct. Notice a few issues and now fixing them. Will add a comment below.